be1-go/cli/organizer/organizer.go

```diff
@@ -20,12 +24,13 @@ func Serve(cliCtx *cli.Context) error {
 	log := be1_go.Logger
 
 	// get command line args which specify public key, port to use for clients
-	// and witnesses
+	// and witnesses, witness' address
 	clientPort := cliCtx.Int("client-port")
 	witnessPort := cliCtx.Int("witness-port")
 	if clientPort == witnessPort {
 		return xerrors.Errorf("client and witness ports must be different")
 	}
+	witness := cliCtx.StringSlice("other-witness")
```

Comment:
Maybe verify that the witness is not empty, just as for the pk in line 36

Reply:
No, I don't think we want to force the use of multiple back ends for now, and even if we wanted to force it, the first server to be launched need to have this field empty to avoid an error to be thrown